terraso_backend/apps/soil_id/models/soil_data.py

```diff
@@ -36,3 +84,15 @@ class SoilData(BaseModel):
         null=True, validators=[MinValueValidator(0), MaxValueValidator(100)]
     )
     slope_shape = models.CharField(null=True, choices=SLOPE_SHAPES)
```

Comment:
suggestion: this part of the spec is a bit confusing, but i think the `down_slope` and `cross_slope` fields should have the `SLOPE_SHAPES` choices, and there is no general `slope_shape` option